tensorflow/python/ops/ctc_ops.py

```diff
@@ -308,6 +314,10 @@ def ctc_greedy_decoder(inputs, sequence_length, merge_repeated=True):
     sequence_length: 1-D `int32` vector containing sequence lengths, having size
       `[batch_size]`.
     merge_repeated: Boolean.  Default: True.
+    blank_index: (Optional). Default: -1. The class index to use for the blank
+      label. Negative values will start from num_classes, ie, -1 will reproduce
+      the ctc_greedy_decoder behavior of using num_classes - 1 for the blank
+      symbol, which, actually, corresponds to the default.
```

Comment:
This docstring is confusing; can you update to more clearly state what the default is? Also, what is this used for, and when would a user want to pass this in? 

Please also add tested examples of using this parameter in the docstring: https://www.tensorflow.org/community/contribute/docs_ref

Reply:
Thank you! I 'll proceed with those changes.